README.md

```diff
@@ -88,8 +88,8 @@ Now, this project is maintained and developed by the [Harbor operator workgroup]
 
 ## Community
 
-* Slack channel `#harbor-operator-dev` at [CNCF Workspace](https://slack.cncf.io)
```

Comment:
[From CI](https://github.com/goharbor/harbor-operator/runs/594276794), please keep dash for unordered list:
```
README.md: 91: MD004/ul-style Unordered list style [Expected: dash; Actual: asterisk]
README.md: 92: MD004/ul-style Unordered list style [Expected: dash; Actual: asterisk]
```

Reply:
This one has been updated in PR.